php/php-mysql-xdevapi/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               php 1.1
+
+name                    php-mysql-xdevapi
+set cap_name            mysql_xdevapi
+categories-append       net devel
+platforms               darwin freebsd openbsd
+maintainers             {ryandesign @ryandesign} openmaintainer
```

Comment:
Since Ryan is being added as maintainer, I want him to approve this.

Reply:
@ryandesign Keeps eye on php, so put him here also.

Reply:
Changed to nomaintainer